images/image-inspector.yml

```diff
@@ -1,7 +1,6 @@
 # <lmeyer> I don't believe anything needs this to be built and released
 # regularly. this is used by cloudforms and the version should correspond to
 # the RPM it installs.  so, let's not re-version and build it.
-mode: disabled
```

Comment:
I think the comment on L1-3 was explaining this `disabled` and can be removed?

Reply:
:heavy_check_mark: